lib/winston/transports/console.js

```diff
@@ -30,7 +31,13 @@ module.exports = class Console extends TransportStream {
     this.name = options.name || 'console';
     this.stderrLevels = this._stringArrayToSet(options.stderrLevels);
     this.consoleWarnLevels = this._stringArrayToSet(options.consoleWarnLevels);
-    this.eol = options.eol || os.EOL;
```

Comment:
I think you might need to either change the behavior of the eol tests that're failing to skip eol, or reimplement the eol for windows compat?

Reply:
Yes, it seems that Windows is using some non-standard line ending?

Reply:
Yes - windows uses `\r\n` instead of `\n` for new line, thus broke the test above